project.clj

```diff
@@ -3,7 +3,7 @@
   :url "http://github.com/shriphani/pegasus"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[clj-http "2.1.0"]
+  :dependencies [[clj-http "3.0.1"]
```

Comment:
Hm `3.0.1` seems be a test version - no idea why it has been released (and not as a SNAPSHOT).